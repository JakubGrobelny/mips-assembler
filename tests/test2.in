# test of all registers (conventional names)

add $zero, $at, $v0
add $v1, $a0, $a1
add $a2, $a3, $t0
add $t1, $t2, $t3
add $t4, $t5, $t6
add $t7, $s0, $s1
add $s2, $s3, $s4
add $s5, $s6, $s7
add $t8, $t9, $k0
add $k1, $gp, $sp
addi $fp, $ra, 0x42