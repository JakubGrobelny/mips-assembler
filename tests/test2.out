00220020
00851820
00e83020
014b4820
01ae6020
02117820
02749020
02d7a820
033ac020
039dd820
23fe0042